sql/src/main/java/org/elasticsearch/action/bulk/SymbolBasedBulkShardProcessor.java

```diff
@@ -273,6 +302,64 @@ private void executeRequests() {
         }
     }
 
+    private void createPendingIndices() {
+        final List<PendingRequest> pendings = new ArrayList<>();
+        final Set<String> indices;
+
+        synchronized (requestsForNewIndices) {
+            indices = ImmutableSet.copyOf(Sets.difference(requestsForNewIndices.keySet(), indicesCreated));
+            for (Map.Entry<String, List<PendingRequest>> entry : requestsForNewIndices.entrySet()) {
+                pendings.addAll(entry.getValue());
```

Comment:
it might be the case that we somehow already created that index, but the requests are still listed there. Those requests must be executed as well. 
we will nonetheless only create the indices from ``indices`` which have the created ones excluded.